en/Obsidian Publish/Introduction to Obsidian Publish.md

```diff
@@ -0,0 +1,16 @@
+---
+alias: [Obsidian Publish]
+---
+
+Obsidian Publish is a cloud-based service where you can publish your notes to the world without leaving your favorite text editor.
```

Comment:
Does "your favorite text editor" mean Obsidian? I get the slight humor but some people might get the wrong idea if their favorite is VS Code or something 🤔

Reply:
Good point! I've removed it for now.